### PR TITLE
[bug] Control+F not working in result from API request

### PR DESCRIPTION
**Pull request by gitstart**
This PR closes https://github.com/hoppscotch/hoppscotch/issues/2946

#### Description
This PR fixes an issue where the `search` or `finder` gets hidden on page scroll. https://github.com/hoppscotch/hoppscotch/issues/2946

#### Before
https://www.loom.com/share/2820b97d4f5b420d8019201c4f126ed0

#### After - Updated
https://www.loom.com/share/355c8d06ab934efc92adf6488ba1d88e

#### Checks
 - [X] My pull request adheres to the code style of this project
 - [ ] My code requires changes to the documentation
 - [ ] I have updated the documentation as required
 - [X] All the tests have passed

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

File: packages/hoppscotch-common/src/helpers/editor/themes/baseTheme.ts [hunk 1]
```diff
@@ -63,6 +63,8 @@ export const baseTheme = EditorView.theme({
   ".cm-panels": {
     backgroundColor: "var(--primary-light-color)",
     color: "var(--secondary-light-color)",
+    zIndex: "1",
+    top: "110px !important",
```
Comment (gitstart) [inline packages/hoppscotch-common/src/helpers/editor/themes/baseTheme.ts:67]: Considering the offset of the nearest scrolling ancestor.
Screenshot 2023-04-17 at 12 34 38

Comment (liyasthomas) [inline packages/hoppscotch-common/src/helpers/editor/themes/baseTheme.ts:67]:
```suggestion
    top: "var(--lower-tertiary-sticky-fold) !important",
```

**Comment by gitstart** [comment]
@liyasthomas other places where the codeMirror is used(e.g `import cURL`) were being affected. Restricted the change to the json response. You can have another look. Thanks.

**Comment by liyasthomas** [comment]
> **Warning**
> Setting the scope just to JSONLenseRederer doesn't seem to be the best way to fix this issue. Fix should be global, should address all CM instances - wherever the search panel is overflowing out of the viewport. Search panels on every CodeMirror instances in the platform needs to be tested across browsers.

> **Note**
> This PR requires my review before merge.

**Comment by liyasthomas** [comment]
Seems there are still few files not covered in this PR which has CodeMirror instances. @gitstart, kindly look into this branch in which I made some changes which covers the remaining files and aligns with our code styles: https://github.com/hoppscotch/hoppscotch/tree/HPS-OSS-1.

I guess I don't have permission to push changes to your branch, so either rebase your branch to the above branch and open a PR or I can go ahead with my branch (https://github.com/hoppscotch/hoppscotch/tree/HPS-OSS-1) and open a new PR.

Till then, I'll mark this PR as draft.

**Pull request closed without merging**